### PR TITLE
remove signup flow

### PR DESCRIPTION
**Pull request by madhavmansuriya40**
https://user-images.githubusercontent.com/51825933/109157928-add3b800-7798-11eb-94f0-3241cc1afb6b.mov

### CODE REVIEW

File: src/app/core/services/tracking.service.ts [hunk 1]
```diff
@@ -63,17 +40,6 @@ export class TrackingService {
     }
   }
 
-  // external APIs
-  onSignin(email, properties) {
```
Comment (Julias0) [inline src/app/core/services/tracking.service.ts:67]: why remove onSignin?

**Review by Julias0 — CHANGES_REQUESTED** [review]
Few comments in tracking service. Add a loom video on what happens when app starts up when not logged in and when logged in

File: src/app/core/services/tracking.service.ts (hunk 1 above)
Comment (madhavmansuriya40) [reply src/app/core/services/tracking.service.ts:67]: by mistake i have done that, but later i added

**Comment by madhavmansuriya40** [comment]
here's loom video link

https://www.loom.com/share/808480ea66de486a9ab49ca4e569c113

File: src/app/core/services/tracking.service.ts [hunk 2]
```diff
@@ -74,15 +60,6 @@ export class TrackingService {
     this.eventTrack('Signin', properties);
   }
 
-  onStateChange(toState, toParams, fromState, fromParams) {
-  }
-
-  eventTrack(action, properties) {
```
Comment (Julias0) [inline src/app/core/services/tracking.service.ts:80]: why remove this?

File: src/app/core/services/tracking.service.ts [hunk 3]
```diff
@@ -74,15 +60,6 @@ export class TrackingService {
     this.eventTrack('Signin', properties);
   }
 
-  onStateChange(toState, toParams, fromState, fromParams) {
```
Comment (Julias0) [inline src/app/core/services/tracking.service.ts:77]: confirm once that this is not being used

**Review by Julias0 — CHANGES_REQUESTED** [review]
illa illa

File: src/app/auth/sign-in/sign-in.page.html [hunk 4]
```diff
@@ -75,7 +75,7 @@
     </div>
     <div class="sign-in--redirect">
       Don't have an account?
-      <a [routerLink]="['/', 'auth', 'sign_up']"> Sign Up</a>
+      <a href="https://www.fylehq.com/"> Sign Up</a>
```
Comment (abhishek1234321) [inline src/app/auth/sign-in/sign-in.page.html:78]: do we need `target="_blank"`?

**Review by abhishek1234321 — APPROVED** [review]

**Pull request closed without merging**